### ISSUE TITLE
Add support for OpenAPI Vendor Extensions

### ISSUE THREAD

**Issue by sheelsh** [ePages-de/restdocs-api-spec#70]
Add OpenAPI Vendor Extensions to add more details to APID docs such as logo, etc.
[OpenAPI 3 Vendor Ext. Ref. ](https://swagger.io/docs/specification/openapi-extensions/) and [ReDoc Ref.](https://github.com/Rebilly/ReDoc/blob/master/docs/redoc-vendor-extensions.md#x-logo)

**Comment by mduesterhoeft** [comment]
I agree that this would be nice - especially for redoc. We could offer a info object extension file or just a list of key value pairs in the plugin configuration. A contribution would be appreciated.
